src/gtl/components/tile-view/tile-view.html

```diff
@@ -1,22 +1,22 @@
 <div class="miq-tile-section">
   <div ng-if="tileCtrl.settings.isLoading" class="spinner spinner-lg"></div>
   <div class="row miq-pagination" ng-if="tileCtrl.rows && tileCtrl.rows.length !== 0">
-    <div ng-if="tileCtrl.rows.length !== 0" class="miq-select-all col-md-3 col-ld-3">
+    <div ng-if="tileCtrl.rows.length !== 0" class="miq-select-all col-md-3 col-ld-3 col-xs-3">
       <label>{{tileCtrl.settings.selectAllTitle}}: </label>
       <input type="checkbox" ng-model="isChecked" ng-click="tileCtrl.onCheckAllTiles(isChecked)" title="{{tileCtrl.settings.selectAllTitle}}" />
     </div>
-    <div class="miq-per-page col-md-2 col-ld-2">
+    <div class="miq-per-page col-md-2 col-ld-2 col-xs-3">
```

Comment:
`xs-2` was too small, and used 2 rows when there was item with label, I'll give it a try and use `col-md-3` and `col-lg-3`.

Yeah my bad about `ld`, I thought bootstrap is consistent at least in medium and large, looks like I was wrong...